src/include/distributed/multi_logical_replication.h

```diff
@@ -26,8 +26,40 @@ extern void LogicallyReplicateShards(List *shardList, char *sourceNodeName,
 									 int sourceNodePort, char *targetNodeName,
 									 int targetNodePort);
 
+extern XLogRecPtr GetRemoteLogPosition(MultiConnection *connection);
+extern List * GetQueryResultStringList(MultiConnection *connection, char *query);
+
+extern void DropShardSubscription(MultiConnection *connection,
+								  char *subscriptionName);
+extern void DropShardPublication(MultiConnection *connection, char *publicationName);
+
+extern void DropShardUser(MultiConnection *connection, char *username);
+extern void DropShardMoveReplicationSlot(MultiConnection *connection,
```

Comment:
Remove 'Move' from function name as it is being used for both Split and Move.

Reply:
Yeah, refactored couple of methods in this codepath. Missed a todo for this. Thanks.